poh/src/poh_service.rs

```diff
@@ -103,7 +103,7 @@ impl PohService {
         hashes_per_batch: u64,
         record_receiver: Receiver<Record>,
     ) -> Self {
-        let poh_exit_ = poh_exit.clone();
+        let poh_exit = poh_exit.clone();
```

Comment:
I don't think there's any reason this needs to be named differently. It's a clone of `poh_exit` which is an `Arc`, so there's no reason to name this differently to avoid dropping the parameter `poh_exit`.